File: backend/conexao.py
import os
import mysql.connector
from mysql.connector import Error
from dotenv import load_dotenv

load_dotenv()

def get_connection():
    try:
        conn = mysql.connector.connect(
            host=os.getenv("DB_HOST", "localhost"),
            user=os.getenv("DB_USER", "root"),
            password=os.getenv("DB_PASSWORD", ""),
            database=os.getenv("DB_NAME", "sgei")
        )
        return conn
    except Error as e:
        print("Falha na conexão:", e)
        return None

def init_db():
    conn = get_connection()
    if conn is None:
        return
    conn.close()

File: dashmin/aluno/conexao.php
<?php

$host = 'localhost';
$user = 'root';
$pass = '';
$db   = 'gestor_escola';

try {
    $pdo = new PDO(
        "mysql:host=$host;dbname=$db;charset=utf8",
        $user,
        $pass,
        [PDO::ATTR_ERRMODE => PDO::ERRMODE_EXCEPTION]
    );
} catch (PDOException $e) {
    die("Falha na conexão: " . $e->getMessage());
}
?>